src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,52 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  animalType = 'hermivore';
```

Comment:
There is a typo in the `animalType` property. It should be 'herbivore' instead of 'hermivore'.